### PR TITLE
Add/remove topics to repository

### LINKED ISSUE #121: Command to update topics for AndcultureCode repositories
Based on the new 'opt-in' requirement for repositories participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/hacktoberfest-update), it would be nice to have command(s) to update (add/remove) topics. 

I have a working prototype for the 'add' functionality in my fork, but we'll want to add the 'remove' functionality too, so we can easily remove the topics once Hacktoberfest is over. They use the same API call ([replaceAllTopics](https://github.com/octokit/rest.js/discussions/1765)), so it's really a matter of adding or filtering out a given topic name. We should be able to consolidate the bulk of the functionality in a private method and expose the public add/remove functions using it.

### PR DESCRIPTION
**Pull request by brandongregoryscott**
Resolves #121 Command to update topics for AndcultureCode repositories

Adds the ability to add/remove topics for a single AndcultureCode repo or all of them at once. Relies on the same personal access token authentication that was already built in - I am assuming only those with permission to update our repo description, settings, etc. will be able to leverage these commands.

Updated the default behavior for the `github` command to require a flag before listing off any repositories, and fixed a bug where the `[and-cli]` prefix was lost in the output

-   [x] Related GitHub issue(s) linked in PR description
-   [x] Destination branch merged, built and tested with your changes
-   [x] Code formatted and follows best practices and patterns
-   [x] Code builds cleanly (no _additional_ warnings or errors)
-   [x] Manually tested
-   [x] Automated tests are passing
-   [x] No _decreases_ in automated test coverage
-   [x] Documentation updated (readme, docs, comments, etc.)
-   [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

**Comment by brandongregoryscott** [comment]
Something is up with the tests - they look like they're timing out in the CI build. I'll dig into it

**Comment by HeyKos** [comment]
@brandongregoryscott I'll review this one for ya.

File: and-cli-github.js [hunk 1]
```diff
@@ -16,23 +20,108 @@ require("./command-runner").run(async () => {
...
         .description(github.description())
+        .option(
+            "--add-topic <topic>",
+            "Add topic to specified repository, or all AndcultureCode repositories if no repo provided"
+        )
+        .option(
+            "--list-topics",
+            "List topics for a given repo (used in conjunction with --repo)"
+        )
+        .option("--list-repos", "Lists all andculture repos")
```
Comment (HeyKos) [inline and-cli-github.js:31]: For consistency consider changing "andculture" to "AndcultureCode".

File: modules/github.js [hunk 2]
```diff
@@ -56,6 +60,50 @@ const github = {
     // #region Public Methods
     // -----------------------------------------------------------------------------------------
 
+    /**
+     * Adds a topic to all AndcultureCode repositories
+     *
+     * @param {string} topic Topic to be added
+     */
+    async addTopicToAllRepositories(topic) {
+        const andcultureRepos = await this.repositoriesByAndculture();
```
Comment (HeyKos) [inline modules/github.js:69]: Consider bailing on the method if the supplied topic is null or an empty string. I think you did something similar in the `addTopicToRepository` call using a call to `_validateTopicInputOrExit`.

File: modules/github.js [hunk 3]
```diff
@@ -237,9 +285,59 @@ const github = {
         }
     },
 
+    /**
+     * Removes a topic from all AndcultureCode repositories
+     *
+     * @param {string} topic Topic to be removed
+     */
+    async removeTopicFromAllRepositories(topic) {
+        const andcultureRepos = await this.repositoriesByAndculture();
```
Comment (HeyKos) [inline modules/github.js:294]: Same here. Consider bailing if a null or empty string is passed in as `topic`.

**Review by HeyKos — APPROVED** [review]
Once you figure out your test running issues, you have my approval. You did a really nice job with your tests here.

File: COMMANDS.md [hunk 4]
```diff
@@ -220,8 +220,12 @@ Various commands to interact with github andculture related resources.
 
 ### Commands
 
--   `and-cli github` - Lists master AndcultureCode repositories
--   `and-cli github -u|--username <username>` - Lists master AndcultureCode repositories as well as those for the supplied github username
+-   `and-cli github -l|--list-repos` - Lists main AndcultureCode repositories
+-   `and-cli github --list-topics --repo AndcultureCode.Cli` - Lists topics for a specified AndcultureCode repo
+-   `and-cli github --add-topic hacktoberfest --repo AndcultureCode.Cli` - Adds a topic to a specified AndcultureCodeRepo
+-   `and-cli github --remove-topic hacktoberfest --repo AndcultureCode.Cli` - Removes a topic from a specified AndcultureCode repo
+-   `and-cli github --remove-topic hacktoberfest` - Removes a topic from all AndcultureCode repos
```
Comment (wintondeshong) [inline COMMANDS.md:227]: 👏 - sliding in the github auth timed out just right haha

File: and-cli-github.js (hunk 1 above)
Comment (wintondeshong) [inline and-cli-github.js:31]: Nitpick: Alphabetize flags

Comment (wintondeshong) [reply and-cli-github.js:31]: I'm also a culprit of switching between 'andculture' and 'andculturecode'. Perhaps we make a constant for CLI and interpolate it throughout the application to settle it?

File: modules/echo.js [hunk 5]
```diff
@@ -12,8 +12,9 @@ const shell = require("shelljs");
 // #region Variables
 // -----------------------------------------------------------------------------------------
 
-const columnLength = 65;
 const { purple, green, red, yellow } = formatters;
+const columnLength = 65;
+const prefix = purple(`[${constants.CLI_NAME}]`);
```
Comment (wintondeshong) [inline modules/echo.js:17]: 👍

**Review by wintondeshong — APPROVED** [review]
Damn @brandongregoryscott awesome work! 👏  I feel like no sooner did I add the github auth stuff did you bang this out.

**Pull request merged**